### ISSUE TITLE
[tools] Add Grafana to environment to monitor server metrics

### ISSUE THREAD

**Issue by landerneRPi** [mdl29/scratchy#127]
Add Grafana to docker compose environment.

References / Examples:
* https://github.com/rycus86/prometheus_flask_exporter/tree/master/examples/sample-signals

**Comment by Draeggon** [comment]
@landerneRPi the link don't redirect to grafana but to prometheus flask exporter

**Comment by landerneRPi** [comment]
> @landerneRPi the link don't redirect to grafana but to prometheus flask exporter

Yes, with an example how to integrate Grafana in docker compose and a dashboard sample in the grafana directory.
